src/objects/Batch_Template__c.object

```diff
@@ -118,6 +132,16 @@
         <trackTrending>false</trackTrending>
         <type>TextArea</type>
     </fields>
+    <fields>
+        <fullName>Display_Total_Prompt__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>When checked, BGE grid prompts user for expected Opportunity total.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>When checked, BGE grid prompts user for expected Opportunity total.</inlineHelpText>
```

Comment:
@smarovich can you review this inline help text for the Display Total Prompt helptext field on the Batch Template page?